ports/qemu/main.c

```diff
@@ -34,7 +34,11 @@
 #include "shared/runtime/gchelper.h"
 #include "shared/runtime/pyexec.h"
 
-#define HEAP_SIZE (100 * 1024)
+#if MICROPY_HEAP_SIZE <= 0
+#error MICROPY_HEAP_SIZE must be a positive integer.
+#endif
+
+#define HEAP_SIZE (MICROPY_HEAP_SIZE * 1024)
```

Comment:
The minimal and zephyr ports specify `MICROPY_HEAP_SIZE` in bytes, so it should also be bytes here, not kibytes.

Reply:
This should be handled in the latest changes I pushed.